### PR TITLE
NLB release to open source from release-1.19 / oci-ccm 0.13.0

### PR DESCRIPTION
**Pull request by l-technicore**
Release to Open Source from internal release-1.19 branch. Includes NLB feature and some other minor fixes and updates.

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -41,6 +41,8 @@ require (
 	github.com/onsi/ginkgo v1.14.1
 	github.com/onsi/gomega v1.10.2
 	github.com/oracle/oci-go-sdk/v31 v31.0.0
+	github.com/oracle/oci-go-sdk/v49 v49.2.0
+	github.com/oracle/oci-go-sdk/v50 v50.1.0
```
Comment (mrunalpagnis) [inline go.mod:43]: keep only 1 version of oci-go-sdk

File: hack/test.sh [hunk 2]
```diff
@@ -23,4 +23,4 @@ export CGO_ENABLED=0
 TARGETS=$(for d in "$@"; do echo ./$d/...; done)
 
 echo "Running tests..."
-GO111MODULE=off go test -coverprofile=coverage.out -v -installsuffix "static" ${TARGETS}
+GO111MODULE=off HTTP_PROXY=http://www-proxy-hqdc.us.oracle.com/ go test -coverprofile=coverage.out -v -installsuffix "static" ${TARGETS}
```
Comment (mrunalpagnis) [inline hack/test.sh]: remove proxy variable

File: pkg/cloudprovider/providers/oci/load_balancer_test.go [hunk 3]
```diff
@@ -17,13 +17,12 @@ package oci
 import (
 	"context"
 	"errors"
+	"github.com/oracle/oci-cloud-controller-manager/pkg/oci/client"
```
Comment (mrunalpagnis) [inline pkg/cloudprovider/providers/oci/load_balancer_test.go]: oraganize imports

File: test/e2e/framework/seclist_util.go [hunk 4]
```diff
@@ -19,7 +19,7 @@ import (
 	"time"
 
 	client "github.com/oracle/oci-cloud-controller-manager/pkg/oci/client"
-	core "github.com/oracle/oci-go-sdk/v31/core"
+	core "github.com/oracle/oci-go-sdk/v49/core"
```
Comment (mrunalpagnis) [inline test/e2e/framework/seclist_util.go]: can be 50 itself

**Review by mrunalpagnis — CHANGES_REQUESTED** [review]
also, remove the changes made in .github folder

File: go.mod (hunk 1 above)
Comment (l-technicore) [reply go.mod:43]: Some old dependencies were remaining in test files, resolved now.

File: pkg/cloudprovider/providers/oci/load_balancer_test.go (hunk 3 above)
Comment (l-technicore) [reply pkg/cloudprovider/providers/oci/load_balancer_test.go]: Done

File: test/e2e/framework/seclist_util.go (hunk 4 above)
Comment (l-technicore) [reply test/e2e/framework/seclist_util.go]: Resolved

**Pull request closed without merging**